content/millennium/r4/base/management/encounter.md

```diff
@@ -98,23 +98,24 @@ _Implementation Notes_
- `_id`        | This or `patient` or `subject` or `account` or `identifier` or `pageContext`   | [`token`]     | The logical resource id associated with the Encounter. Example: `7891`
+ `_id`        | This or `patient` or `subject` or `account` or `identifier` or `pageContext`   | [`token`]     | The logical resource ID associated with the encounter. Example: `7891`
 `patient`     | This or `subject` or `account` or `_id` or `identifier` or `pageContext`       | [`reference`] | The patient present at the encounter. Example: `12345`
 `subject`     | This or `patient` or `account` or `_id` or `identifier` or `pageContext`       | [`reference`] | The patient present at the encounter. Example: `subject=Patient/12345` or `subject:Patient=12345`
 `account`     | This or `patient` or `subject` or `_id` or `identifier` or `pageContext`       | [`reference`] | The account associated with the encounters. Example: `F703726`
 `identifier`  | This or `patient` or `subject` or `account` or `_id` or `pageContext`          | [`token`]     | An encounter's identifier. Example: `urn:oid:1.2.243.58|110219457`
 `date`        | No                                                                             | [`dateTime`]  | Datetime range into which the encounter's period datetime falls. Must be prefixed by 'ge', 'gt', 'le' or 'lt'.
 [`_count`]    | No                                                                             | [`number`]    | The maximum number of results to return.
 `status`      | No                                                                             | [`token`]     | The status of the encounter. Example: `planned`
-`_revinclude` | No                                                                             | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
+`_revinclude` | No                                                                             | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example: `_revinclude=Provenance:target`
   
 Notes:
 
-* `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
-* `_revinclude` parameter may be provided with the `_id/patient/subject/account` parameter. Example: `_id=74771957,4732066&_revinclude=Provenance:target`
-* `_revinclude` is provided in a request to the closed endpoint, the OAuth2 token must include the `user/Provenance.read` scope. Currently, `patient/Provenance.read` is not supported and hence `_revinclude` cannot be utilized for patient persona.
+* When using the `_revinclude` parameter:
+  * It may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
+  * It may be provided with the `_id/patient/subject/account` parameter. Example: `_id=74771957,4732066&_revinclude=Provenance:target`
```

Comment:
let's split these fields to be more clear that they are separate parameters

Reply:
Do you mean split like this? 

* When using the `_revinclude` parameter:
  * It may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
  * It may be provided with the `_id parameter. Example: `_id=74771957,4732066&_revinclude=Provenance:target`
  * It may be provided with the `patient` parameter. Example: <example here>
  * It may be provided with the `subject` parameter. Example: <example here> 
  * It may be provided with the `account` parameter. Example: <example here>